samples/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@google-cloud/talent-samples",
+  "description": "Samples for the Cloud Talent Solution client library for Node.js",
+  "license": "Apache-2.0",
+  "author": "Google LLC",
+  "engines": {
+    "node": ">=8"
+  },
+  "repository": "googleapis/nodejs-talent",
+  "private": true,
+  "files": [
+    "*.js"
+  ],
+  "scripts": {
+    "cleanup": "node scripts/cleanup",
+    "test": "mocha system-test/*.js --timeout 600000"
```

Comment:
I want to get away from `system-test` - can you just make this say `mocha` for now?